src/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -220,6 +221,10 @@ public static void main(final String[] args) {
         snpCounter   = new GenotypeConcordanceCounts();
         indelCounter = new GenotypeConcordanceCounts();
 
+        //need to know size of intervals to add missing-missing sites for NIST schema.
+        long intervalBaseCount = 0L;
+        if(usingIntervals && MISSING_SITES_HOM_REF) {intervalBaseCount = intervals.getBaseCount();}
```

Comment:
I'd recommend putting this all in one line:
final long intervalBaseCount = intervals.getBaseCount();
You don't need the if statement here to protect the variable initialization since it's only used below within the exact same if statement.  (And, come to think about it, you could just move this line down below to where it is used)